src/types.ts

```diff
@@ -40,19 +40,22 @@ export interface LookerAuthConfig {
 
 /**
  * Cookieless request init
+ * Looker 22.20
  */
 export interface CookielessRequestInit extends RequestInit {
   url: string
 }
 
 /**
  * Cookieless request callback function
+ * Looker 22.20
```

Comment:
Do you want to use `Looker 22.20+` notation here?

Reply:
I followed the pattern used elsewhere. Does not use the +. I might create a separate PR just to fix that. Leaving as is for now